### PR TITLE
docs: add workspace classes section

### PR DESCRIPTION
**Pull request by stefanicjuraj**
Once the deployment of the https://github.com/daytonaio/enterprise-docs/issues/124 is released, it can be added in this new section.

### CODE REVIEW

File: src/content/docs/administration/workspace-classes.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+- **Name**
+
+    The name of the Workspace class.
+
+    Example: `My Workspace Class`
+
+- **CPU**
+
+    Specify the CPU resource limit allocated to Workspaces within this class.
```
Comment (ivan-burazin) [inline src/content/docs/administration/workspace-classes.mdx:29]: @stefanicjuraj  - Specify the number of CPUs allocated to Workspaces within this class.

File: src/content/docs/administration/workspace-classes.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+    Example: `2`
+
+- **CPU Overprovisioning**
+
+    Optionally, specify the percentage of overprovisioning for the CPU resource request. Overprovisioning allows more resources to be allocated than available, relying on the assumption that not all resources will be used simultaneously.
+
+    Example: `200%`
```
Comment (ivan-burazin) [inline src/content/docs/administration/workspace-classes.mdx:37]: @fabjanvucina is it C% or mutiple of X?

File: src/content/docs/administration/workspace-classes.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    Define the memory resource limit (in GB) for the Workspace.
+
+    Example: `4`
+
+- **Memory Overprovisioning**
+
+    Optionally, specify the percentage of overprovisioning for the memory resource request, similar to CPU overprovisioning.
+
+    Example: `200%`
```
Comment (ivan-burazin) [inline src/content/docs/administration/workspace-classes.mdx:49]: @vedranjukic can we do different overprovitionef of ram and CPU? so Ram 2x and cpu 4x?

File: src/content/docs/administration/workspace-classes.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+    Specify the storage resource limit (in GB) assigned to the Workspace.
+
+    Example: `50`
+
+- **Runtime Class**
+
+    Optionally, specify the container runtime configuration, such as `sysbox-runc`, which manages the execution environment of the Workspace.
+  
+    Example: `sysbox-runc`
```
Comment (ivan-burazin) [inline src/content/docs/administration/workspace-classes.mdx:61]: @vedranjukic this should be a drop down, with just the Runtimes that are supported.

File: src/content/docs/administration/workspace-classes.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+While overprovisioning can significantly increase the efficiency and cost-effectiveness of resource usage in a cluster, it must be carefully managed to avoid potential drawbacks, such as unexpected Workspace eviction or performance issues due to resource contention.
+
+## Remove a Workspace Class
+
+Daytona provides the option to remove a Workspace class if it is no longer required.
+
+1. Click the bin icon next to the Workspace class you want to remove.
+
+:::danger
```
Comment (ivan-burazin) [inline src/content/docs/administration/workspace-classes.mdx:93]: @stefanicjuraj should danger not be red?

File: src/content/docs/administration/workspace-classes.mdx [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+After removing the Workspace class, Workspaces associated with the class will be affected. Ensure that the Workspace class is no longer in use before proceeding with the removal.
```
Comment (ivan-burazin) [inline src/content/docs/administration/workspace-classes.mdx:94]: @fabjanvucina will this destroy all WS that were created in that class? if so this should be stated as such

File: src/content/docs/administration/workspace-classes.mdx [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  
+    Example: `sysbox-runc`
+
+- **GPU**
+
+    Optionally, specify the GPU resource limit, used for Workspaces requiring GPU computation.
+
+    Example: `1`
+
+## Overprovisioning
```
Comment (ivan-burazin) [inline src/content/docs/administration/workspace-classes.mdx:69]: @stefanicjuraj would probably put over provisioning before create/delete?

File: src/content/docs/administration/workspace-classes.mdx (hunk 5 above)
Comment (stefanicjuraj) [reply src/content/docs/administration/workspace-classes.mdx:93]: This commit will reflect all danger/caution note changes to red: https://github.com/daytonaio/enterprise-docs/pull/137/commits/22c2fdec8b4c1a78e93c4add8ce5129e1575d781

**Review by ivan-burazin — APPROVED** [review]

**Review by ivan-burazin — APPROVED** [review]

File: src/content/docs/administration/workspace-classes.mdx (hunk 3 above)
Comment (vedranjukic) [reply src/content/docs/administration/workspace-classes.mdx:67]: yes

**Review by ivan-burazin — APPROVED** [review]

**Review by fabjanvucina — APPROVED** [review]

**Comment by fabjanvucina** [comment]
Since @vedranjukic is listed as a reviewer, we will wait for his final comments before merging this PR.

**Pull request merged**